raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -208,8 +209,9 @@ public KafkaRaftClient(
         int fetchMaxWaitMs,
         OptionalInt nodeId,
         LogContext logContext,
-        Random random
-    ) {
+        Random random,
+        RaftConfig raftConfig
+    ) throws IOException {
```

Comment:
Is anything in here throwing `IOException`?

Reply:
Yea. This is an artifact from the `quorumState.initialize` change. Since that's moved down to the `client.initialize`, we can remove it.